Demo/Demo/SwiftUIComponents/PayPalWebViews/PayPalWebOrderCompletionResultView.swift

```diff
@@ -0,0 +1,53 @@
+import SwiftUI
+
+struct PayPalWebOrderCompletionResultView: View {
```

Comment:
The names of these classes confuse me a bit. What is the difference b/w `PayPalWebOrderCompletionResultView` & `PayPalWebOrderCompletionView`?

Reply:
I have this view (`PayPalOrderCompletionView`) where eI carry over order approval results to second screen to display authorize/capture results (`PayPalWebOrderCompletionResultView`)
![Simulator Screenshot - iPhone 15 Pro - 2023-11-13 at 11 25 48](https://github.com/paypal/paypal-ios/assets/9273272/89e33f88-9900-421d-bab9-6440e914d559)
![OrderCompletionResultViewPic](https://github.com/paypal/paypal-ios/assets/9273272/1fc9e878-50ce-487d-85a8-a7a7d993de11)

Reply:
There are several results in these flows. For PayPalWeb, I have order creation result, approval result and the order completion result (result of authorize/capture). So I am distinguishing them with the names.